controllers/operator/suite_test.go

```diff
@@ -55,8 +59,29 @@ import (
 )
 
 const (
-	testNamespace     = "kyma-system"
+	testNamespace    = "kyma-system"
+	apiGatewayCRName = "default"
+
 	eventuallyTimeout = time.Second * 10
+
+	kymaNamespace   = "kyma-system"
+	kymaGatewayName = "kyma-gateway"
+
+	controlledList = `
```

Comment:
Yes, I do not like this. It is only for the tests. I can not make it work for the `testenv` tests since they are running in some virtual KCP env and I did not find a way to provide the file available for this controller container env. If you know how, I would be happy to get rid of that.

Reply:
Ok, I get it. Let's keep it like this for now.